Assets/Mirror/Tests/CustomWriterTest.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using NUnit.Framework;
+
+namespace Mirror.Tests
+{
+    public class CustomWriterTest
+    {
+        public class MyClass
+        {
+            public string name {get; set; }
```

Comment:
missing a space between `{` and `get;`

Reply:
why is this a property anyways? is this necessary for the test to work? Document it if so, or add a second test with a bare field if not

Reply:
Yes,   it is necessary for the test to actually test it.  
If I make it a field,  mirror will generate the proper serializer/deserializer even without the [Reader]/[Writer]  stuff